src/views/Cookies/Cookies.module.css

```diff
@@ -0,0 +1,53 @@
+.mainWrap {
+  place-content: center;
+  flex: 1;
+  padding-top: 80px;
+  padding-bottom: 80px;
```

Comment:
maybe we need some TODOS for ui library to store values to then implement consistent paddings for all pages

Reply:
Good idea, but I'm not sure if it should be a ui kit or a component in the ninja